awscli/customizations/s3/subcommands.py

```diff
@@ -818,9 +789,46 @@ class RbCommand(S3TransferCommand):
         "deleted."
     )
     USAGE = "<S3Uri>"
-    ARG_TABLE = [{'name': 'paths', 'nargs': 1, 'positional_arg': True,
+    ARG_TABLE = [{'name': 'path', 'positional_arg': True,
                   'synopsis': USAGE}, FORCE]
 
+    def _run_main(self, parsed_args, parsed_globals):
+        super(RbCommand, self)._run_main(parsed_args, parsed_globals)
+
+        if not parsed_args.path.startswith('s3://'):
+            raise TypeError("%s\nError: Invalid argument type" % self.USAGE)
+        bucket, key = split_s3_bucket_key(parsed_args.path)
+
+        if parsed_args.force:
+            # TODO: always validate or never validate
+            # This is currently placed here to preserve behavior with the
+            # previous implementation which only validated key existence when
+            # --force is provided.
+            if key:
+                raise ValueError('Please specify a valid bucket name only.'
+                                 ' E.g. s3://%s' % bucket)
+            self._force(parsed_args.path, parsed_globals)
```

Comment:
Capturing offline discussion: We should keep the rc since we document it, but we'll prepend `remove_bucket failed:` to the error message.